tracer/dependabot/Datadog.Dependabot.Integrations.csproj

```diff
@@ -232,8 +232,8 @@
 
     <!-- Integration: Microsoft.VisualStudio.TestPlatform.MSTest.TestAdapter -->
     <!--    Assembly: Microsoft.VisualStudio.TestPlatform.MSTest.TestAdapter -->
-    <!-- Latest package https://www.nuget.org/packages/MSTest.TestAdapter/3.7.3 -->
-    <PackageReference Include="MSTest.TestAdapter" Version="3.7.3" />
+    <!-- Latest package https://www.nuget.org/packages/MSTest.TestAdapter/3.8.0 -->
+    <PackageReference Include="MSTest.TestAdapter" Version="3.8.0" />
```

Comment:
This breaks all the MSTest tests apparently :sigh:

Reply:
Pushed a commit that removes the MStest bump, so that we can merge the other changes